app/components/TuneCard/index.js

```diff
@@ -0,0 +1,102 @@
+/**
+ *
+ * TuneCard
+ *
+ */
+
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Card, Typography, Image, Button } from 'antd';
+import { PlayCircleFilled, PauseCircleFilled } from '@ant-design/icons';
+
+const ItemCard = styled(Card)`
+  && {
+    margin: 20px 0;
+    display: flex;
+    width: ${(props) => props.maxwidth}px;
+    flex-direction: column;
+    justify-content: space-around;
+  }
+`;
+
+const iconStyle = { fontSize: 32 };
+
+const IconButton = styled(Button)`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+export function TuneCard({
+  maxwidth,
+  artistName,
+  collectionName,
+  cardImg,
+  previewUrl,
+  currentTrack,
+  setCurrentTrack,
+  isTrackPlaying,
```

Comment:
why don't we delegate the playing/pausing of the track to the component itself. Looks like we're controlling it based on some prop values that are passed from outside.